fgpyo/util/metric.py

```diff
@@ -233,6 +233,10 @@ def format_value(cls, value: Any) -> str:
                 return ""
             else:
                 return "{" + ",".join(cls.format_value(v) for v in value) + "}"
+
+        if value is None:
+            return ""
+
```

Comment:
```suggestion
```

Reply:
@nils removing this makes it so that `None` will no longer be formatted as `""`. Is that the intention and I have massively misunderstood something. Or is it in the incorrect place?

Reply:
@nils am I correct in assuming this is meant to tell me those lines are not placed correctly in the function (ie move them lower)?

Reply:
I moved those lines lower in commit [6fad7c9](https://github.com/fulcrumgenomics/fgpyo/commit/6fad7c9bfe4b3332f8de3c340c7bba6e2d62ee58)